sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -41,9 +41,14 @@ dependencies {
   compile library.java.google_auth_library_credentials
   compile library.java.google_auth_library_oauth2_http
   compile library.java.google_cloud_bigquery_storage
-  compile library.java.google_cloud_bigtable_client_core
+  compile(library.java.google_cloud_bigtable_client_core) {
```

Comment:
Why are these exclusions needed?

The maven pom that this module will generate has grpc-core and grpclb as top level dependencies and maven resolution rules will choose those over the ones specified by bigtable-client-core or google-cloud-core-grpc so I would have expected that we wouldn't need the exclusions.

Reply:
These exclusions are for Maven's version range constraint. The dependency was transitively requesting `io.grpc:grpc-core:jar:[1.26.0,1.26.0]`, while another dependency requests `io.grpc:grpc-core:jar:[1.25.0,1.25.0]`. Detail in https://issues.apache.org/jira/browse/BEAM-8691?focusedCommentId=17025456&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17025456

Reply:
I wasn't aware that the build would fail before the dependency mediation kicked in.